app/controllers/articles_controller.rb

```diff
@@ -62,7 +62,7 @@ def set_article
   end
 
   def list_articles
-    articles = Article.with_attached_thumbnail.includes(user: { avatar_attachment: :blob }).order(created_at: :desc).page(params[:page])
+    articles = Article.with_attached_thumbnail.includes(user: { avatar_attachment: :blob }).order(published_at: :desc).page(params[:page])
```

Comment:
[NITS]

動作確認の
> [ブログ一覧ページ](http://localhost:3000/articles)で作成したWIP記事がブログ記事の先頭に存在されていること

についてですが、既存の「仮想マシンにDebianをインストールしよう」が常に先頭に来るようになっています。


![image](https://github.com/fjordllc/bootcamp/assets/111285341/b51e3d11-c618-481f-8c6d-bee5064e6bb6)

`created_at`も第二引数に渡せばWIPの中でも最新順に並べられるのかなと思いました。


```suggestion
    articles = Article.with_attached_thumbnail.includes(user: { avatar_attachment: :blob }).order(published_at: :desc, created_at: :desc).page(params[:page])
```

ただ開発環境だけの問題かもしれませんし、WIPが沢山あってどれが最新か分かりづらい、という状況も実際はほぼ起こらないのかなぁとも思います。

(仕様次第だとも思いますし、)ご判断はお任せします！

Reply:
@masyuko0222

レビューありがとうございます！

> [ブログ一覧ページ](http://localhost:3000/articles)で作成したWIP記事がブログ記事の先頭に存在されていること

確かに仰る通りで、先頭にはなっていないですよね💦
ご指摘ありがとうございます🙏
「WIP記事群に作成したWIP記事が含まれていること」が本来言いたかったことなので、コメントを修正しますね！

また、[仕様の擦り合わせ](https://github.com/fjordllc/bootcamp/issues/7940#issuecomment-2216503489)から「一度公開してWIPにした記事の扱い」をどう判断したか？についても補足しようと思います🙇